src/mscorlib/shared/System/Numerics/Vector.cs

```diff
@@ -763,6 +767,36 @@ private Vector(ref Register existingRegister)
         {
             this.register = existingRegister;
         }
+
+        /// <summary>
+        /// Constructs a vector from the given span. The span must contain at least Vector'T.Count elements.
+        /// </summary>
+        [Intrinsic]
```

Comment:
Why do we need this attribute? `Vector<T>` intrinsics are not recognized by the new-style intrinsic system.

Reply:
BTW, this new constructor is just a software implementation rather than intrinsic.

Reply:
Removed...